src/main/java/pl/datamatica/traccar/model/Device.java

```diff
@@ -692,6 +693,8 @@ public void setCustomIconId(Long value) {
     }
     
     @Temporal(TemporalType.DATE)
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ",
```

Comment:
Czy ta zmiana nie popsuje obecnie znajdującego się w bazie formatu?
Czy ma to tylko wpływ na JSON-a?